master/getting-started/openstack/installation/redhat.md

```diff
@@ -57,29 +57,45 @@ networking.
 
 ### Configure YUM repositories
 
-{% include ppa_repo_name %}
-
-Add the EPEL repository -- see <https://fedoraproject.org/wiki/EPEL>.
-You may have already added this to install OpenStack.
-
-Configure the {{site.prodname}} repository:
-
-```
-    cat > /etc/yum.repos.d/calico.repo <<EOF
-    [calico]
-    name=Calico Repository
-    baseurl=https://binaries.projectcalico.org/rpm/{{ ppa_repo_name }}/
-    enabled=1
-    skip_if_unavailable=0
-    gpgcheck=1
-    gpgkey=https://binaries.projectcalico.org/rpm/{{ ppa_repo_name }}/key
-    priority=97
-    EOF
-```
+1. From a terminal prompt, create a new file called bird.repo in the 
+   `/etc/yum.repos.d/` directory.
+
+   ```bash 
+   vi /etc/yum.repos.d/bird.repo
+   ```
+   
+   > **Note**: Creating a file in this directory requires root privileges.
+   {: .alert .alert-info}
+
+1. Add the following lines to the file.
+
+   ```
+   [bird]
+   name=Network.CZ Repository
+   baseurl=ftp://repo.network.cz/pub/redhat/
+   enabled=1
+   gpgcheck=0
+   gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-network.cz
+   ```
+
+1. Save and close the file.
+
+1. Use the following command to install BIRD.
+
+   ```bash
+   yum install -y bird
+   ```
+   
+> **Note**: We do not recommend installing [Extra Packages for Enterprise Linux (EPEL)](https://fedoraproject.org/wiki/EPEL). 
+> EPEL lacks official Red Hat support. It contains the BIRD package, but it also contains 
+> other packages. Installing EPEL may cause existing packages on your system to
+> be overwritten with unsupported packages. To avoid issues of this kind, we recommend 
+> using the method described above.
+{: .alert .alert-info}
```

Comment:
OK, I got that about half wrong....  d'oh!

In fact we *can* use a vanilla BIRD on an OpenStack compute node, because with OpenStack we don't document or advise anyone to use IP-in-IP.  And in our own testing we use a vanilla BIRD without the IP-in-IP support.

So that means that what we should have here is:

- instructions for the upstream BIRD repo - as you've drafted, plus downloading the GPG key

- **and** keep the instructions as they were for the Calico repo.

(But should be fine to delete the EPEL install instruction.)

In an ideal world, we should also **test** that this really works :-)  I'm not sure I can absorb that right now, but perhaps it would make sense for you to create a backlog item for that.